libs/application/templates/children-residence-change/src/dataProviders/MockNationalRegistryProvider.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Application,
+  BasicDataProvider,
+  FailedDataProviderResult,
+  SuccessfulDataProviderResult,
+} from '@island.is/application/core'
+
+import { PersonResidenceChange } from '@island.is/application/templates/children-residence-change'
+import { NationalRegistry, DataProviderTypes, CRCApplication } from '../types'
+
+export class MockNationalRegistryProvider extends BasicDataProvider {
+  readonly type = DataProviderTypes.MOCK_NationalRegistry
+
+  async provide(application: Application): Promise<NationalRegistry> {
+    const query = `
+      query NationalRegistryUserQuery {
+        nationalRegistryUser {
+          nationalId
+          fullName
+          address {
+            code
+            postalCode
+            city
+            streetAddress
+            lastUpdated
```

Comment:
Even though it is not ready don't we want the query to reflect what data we expect?